dataset/dictionary/system.ts

```diff
@@ -1127,12 +1135,93 @@ export default [
     pronunciationJa: "ねっとうのうたげ",
     tags: [ "natlan" ],
   },
+
+  //
+  // Nod-Krai
+  //
+  {
+    en: "The Wild Hunt",
+    ja: "ワイルドハント",
+    zhCN: "狂猎",
+    zhTW: "狂獵",
+    notes: "アビスに冒された亡霊が出現する現象",  // notes:「ワイルドハント」は集団ではなく現象という説明があるためorganizationではないと判断
+    tags: [ "nodkrai" ],
```

Comment:
1. Let's write commemt in English.

2. It looks like a comment for the `notes` while it explains about `tags`. Let's move it after the line of `tags`.

3. (Optional) `enemy` tag might be another candidate, I guess. (I leave a decision up to you.)

```suggestion
    notes: "アビスに冒された亡霊が出現する現象",
    tags: [ "nodkrai" ], // The Wild Hunt is not an organization but a phenomenon or disaster, as explained in the game.
```